src/gsconnect-preferences

```diff
@@ -1,104 +1,19 @@
-#!/usr/bin/env -S gjs -m
+#!/usr/bin/sh
```

Comment:
I don't think that's guaranteed to exist at that path.

...is what I wanted to write, and then I made the mistake of going to actually look at what POSIX says, and now I'm just sad.

![image](https://github.com/user-attachments/assets/4f4a4780-3daa-4770-b224-65af9ae3ff7f)

Reply:
On my Ubuntu system here both `/bin/sh` and `/usr/bin/sh` are symlinks to `dash`. I once set my login shell to `/usr/bin/bash` on a `/usr` merged Arch years ago and couldn't log in after that because `/bin/bash` was the canonical name even though both could execute it. I'd guess `/bin/sh` is perhaps slightly more safe.

I think an executable file with no shebang will get passed to `sh` https://utcc.utoronto.ca/~cks/space/blog/unix/EmptyFileWhyTrue, but that's also quite a silly way to do it, and then we'd perhaps need to have a non-shebang ya rly comment stating what we are doing.